### PR TITLE
feat: add has_root_access to netapp vol export rules

### PR DESCRIPTION
**Pull request by BrendanThompson**
This adds the ability to set `has_root_access` on `export_policies`.

### CODE REVIEW

**Comment by BrendanThompson** [comment]
@katbyte @tombuildsstuff – any chance that I could get a review here? Keen to get this merged in ASAP 😃.

**Review by katbyte — DISMISSED** [review]
Thanks for the pr @BrendanThompson - aside from one comment about the schema this looks good. Could we change `has_root_access` to `root_access_enabled` to be consistent with the rest of the provider?

**Comment by BrendanThompson** [comment]
@katbyte I have updated that as requested. Thanks 😊

**Comment by BrendanThompson** [comment]
@katbyte – sorry I forgot to add this in everywhere -facepalm- I have pushed that change up now.

**Comment by BrendanThompson** [comment]
@katbyte Will this change actually be released in this Friday's release?

File: website/docs/r/netapp_volume.html.markdown [hunk 1]
```diff
@@ -140,6 +140,8 @@ An `export_policy_rule` block supports the following:
 
 * `unix_read_write` - (Optional) Is the file system on unix read and write?
 
+* `allow_root_access` - (Optional) Has root access to the volume?
```
Comment (tombuildsstuff) [inline website/docs/r/netapp_volume.html.markdown:143]: presumably this should be:

```suggestion
* `allow_root_access` - (Optional) Is root access permitted to this volume?
```

**Review by tombuildsstuff — CHANGES_REQUESTED** [review]
hey @BrendanThompson 

Thanks for pushing those changes, I've left one comment around the documentation inline but this should otherwise be good to go 👍 

Thanks!

**Comment by BrendanThompson** [comment]
Hey @tombuildsstuff , thanks for that. I've updated as per the suggestion.

**Review by tombuildsstuff — APPROVED** [review]
LGTM - thanks for pushing that change @BrendanThompson

**Pull request merged**